### DISCUSSION TITLE
merlin AX82U error: unexpected ")"

### DISCUSSION THREAD

**Discussion by jagernb** [messense/aliyundrive-webdav#516] [Q&A]
![image](https://user-images.githubusercontent.com/105163570/179246903-447818b2-6852-4bd1-b843-bad0f906b24f.png)
2022年07月15日 22:43:04: 当前已进入aliyundrivewebdav_config.sh
2022年07月15日 22:43:04: 当前已进入aliyundrivewebdavconfig.sh执行
2022年07月15日 22:43:04: 先结束进程
2022年07月15日 22:43:04: 参数为：8082 -r ad08ba99411747d888c4fecf58b0e45c --root / -S 10485760 --cache-size 1000 
2022年07月15日 22:43:09: aliyundrive 进程启动失败！请检查参数是否存在问题，即将关闭
2022年07月15日 22:43:09: 失败原因：
2022年07月15日 22:43:09: error: unexpected ")"

**Comment by sjjsjss** [comment]
同样的问题，显示未启动阿里云盘
